lib/DxcSupport/HLSLOptions.cpp

```diff
@@ -604,17 +604,17 @@ int ReadDxcOpts(const OptTable *optionTable, unsigned flagsToInclude,
     opts.ScanLimit = std::stoul(std::string(limit));
 
   for (std::string opt : Args.getAllArgValues(OPT_opt_disable))
-    opts.DxcOptimizationToggles[llvm::StringRef(opt).lower()] = false;
+    opts.OptToggles.Toggles[llvm::StringRef(opt).lower()] = false;
```

Comment:
Should we validate that these options should map to a known toggle and issue a warning if they don't? Do we expect there is a valid use case for passing a string that does not map to a known toggle?

Reply:
I think with this change, the most we could do here is to add a warning, and even that might be too much of a functional change. We should have some more discussion about that.